ProjectSidewalk/SidewalkWebpage
# 2958: Automatically showing validate label metadata for Amsterdam

Resolves #2958 

People in Amsterdam requested to have the validate label metadata (description and severity) shown automatically without needing to hover. I implemented this by editing the validation html file to make Amsterdam-specific html code that shows the label description box and delete button automatically rather than starting out as hidden. I also made sure to set the positions of the two elements when the label is loaded. Then, I added coded to stop any code controlling the hover feature if the current city is Amsterdam, and included label description box in the hide and unhide label feature.

Testing instructions
1. Starting on Amsterdam, start validating and check that:
- the label description box and hide/unhide button are showing automatically and are positions on the bottom left corner and top corner of the label
- the hide/unhide button is hiding both the label itself and the description box leaving behind a dotted outline for the label marker
- other buttons are working (agree, disagree, etc.) the way they should be
2. Move to another city, start validating and check that:
- the label description box and hide/unhide button are starting as hidden and show up when hovered over; the positions should be the exact same as what is described above
- All other buttons are working the way they should be (agree, disagree, hide/unhide button, etc.)
